furniro-front-end/src/app/header/Index.ts

```diff
@@ -0,0 +1,6 @@
+// Header components
+import Header from "./components/Header";
+// Buttons and Icons
+import HeaderButtons from "./components/HeaderButtons";
+
+export { HeaderButtons, Header };
```

Comment:
I meant like this: 
// Components
import Header from "./components/Header";
import HeaderButtons from "./components/HeaderButtons";